modules/UI/util/MessageHandler.js

```diff
@@ -16,7 +16,7 @@ const logger = Logger.getLogger(__filename);
  * Flag for enabling/disabling popups.
  * @type {boolean}
  */
-let popupEnabled = true;
+const popupEnabled = true;
```

Comment:
Delete this and adjust all conditions which read the value, as it's no longer set. Also our code used to set it to false, so that should be the new behavior now.